crypto/signature.go

```diff
@@ -159,3 +160,16 @@ func verifySignature(pubKeyEntries openpgp.EntityList, origText io.Reader, signa
 
 	return nil
 }
+
+func (sig *PGPSignature) getCreationTime() (int64, error) {
+	packets := packet.NewReader(bytes.NewReader(sig.Data))
+	p, err := packets.Next()
+	if err != nil {
+		return 0, goerrors.Wrap(err, "gopenpgp: Can't parse signature")
+	}
+	sigPacket, ok := p.(*packet.Signature)
+	if !ok {
+		return 0, errors.New("gopenpgp: non signature packet found")
```

Comment:
what about returning `-1` (or the max int64 value explicitly)? since 0 is a valid timestamp, but -1 would be in the future

Reply:
In general, in go it's kind of standard to return zero values when also returning an error, and the caller is supposed to always check the error before using the value (and the value is in that sense meaningless if there is an error), so I think it's fine tbh

Reply:
And in apps it will throw an exception so the value won't be reachable anyway